index.html

```diff
@@ -786,6 +786,8 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
 initially 0.</li>
 <li>A <dfn>resource timing buffer full flag</dfn> which is
 initially false.</li>
+<li>A <dfn>resource timing secondary buffer current size</dfn> which is
+initially 0.</li>
 <li>A <dfn>secondary entry buffer</dfn> to store <a>PerformanceEntry</a>
```

Comment:
nit: inconsistent dfns. One includes 'resource timing', like the rest, the other does not.

Reply:
Also inconsistency between 'secondary buffer' and 'secondary entry buffer'

Reply:
ok